Neil's suggestions:  ignore unrecognized JSON names, and longer expla… (#52)
ietf-wg-dnsop/draft-ietf-dnsop-structured-dns-error: draft-ietf-dnsop-structured-dns-error.md

  * Number: 5

>   * Meaning: Network Operator Policy. The code indicates that the request was filtered according to policy determined by the operator of the local network (where local may be a relative term, i.e., it could refer to Local Area Network, or to the network of the ISP selected by the user).

Reviewer: ```suggestion
  * Meaning: Network Operator Policy. The code indicates that the request was filtered according to a policy imposed by the operator of the local network (where local network is a relative term, e.g., it may refer to a Local Area Network or to the network of the ISP selected by the user).
```

  * Number: 6

>   * Meaning: DNS Operator Policy. The code indicates that the request was filtered according to policy determined by the operator of the DNS server. This would be different from Network Operator Policy when a third-party or “over-the-top” DNS resolver is used.

Reviewer: ```suggestion
  * Meaning: DNS Operator Policy. The code indicates that the request was filtered according to policy determined by the operator of the DNS server. This is different from the "Network Operator Policy" code when a third-party DNS resolver is used.
```

Third-party is generic enough
